source/OpenBVE/Game/Events/EventTypes/Station.cs

```diff
@@ -145,9 +133,8 @@ public override void Trigger(int Direction, EventTriggerType TriggerType, TrainM
 							{
 								Train.StationState = TrainStopState.Pending;
 							}
-							
-							int d = Train.DriverCar;
-							Program.Sounds.StopSound(Train.Cars[d].Sounds.Halt.Source);
+
+							Program.Sounds.StopSound(Train.SafetySystems.PassAlarm.Sound);
```

Comment:
Needs a refactor, so that we call a halt method on the pass alarm instead.

Will then allow future abstraction of the pass alarm for more interesting implementations.